### PR TITLE
build(refactor): cleanup scripts folder

### PR DESCRIPTION
**Pull request by brainbicycle**
This PR resolves [PHIRE-324] 

### Description

Organizes our scripts folder a bit before trying to optimize a bit, now organized into the following folders:

```
/changelog
/check-flags
/ci
/city-guide
/codepush
/deploys
/detect-secrets
/helpers
/hermes
/palette
/setup
/utils
```

Deletes a few unused scripts.

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- clean up scripts folder - brian 

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[PHIRE-324]: https://artsyproduct.atlassian.net/browse/PHIRE-324?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 4768f2b1fd4089f3175486ec63ebe2f71a2861c0
</p>

File: .gitignore [hunk 1]
```diff
@@ -83,8 +83,9 @@ eslint-errors.json
 # typescript
 tsc_raw.log
 
-# query map
+# data folder
 data/complete.queryMap.json
+data/PreHeatedGraphQLCache/*
```
Comment (gkartalis) [inline .gitignore:88]: 👀

**Review by gkartalis — APPROVED** [review]
This is SO GOOD 👏

File: .gitignore (hunk 1 above)
Comment (gkartalis) [reply .gitignore:88]: didn't even know that this existed

Comment (brainbicycle) [reply .gitignore:88]: It is news to me too 😬 used to be used for city guide, wondering if things are slower since we stopped using this

**Pull request merged**